Libplanet/Net/BoundPeer.cs

```diff
@@ -53,6 +54,53 @@ private BoundPeer(SerializationInfo info, StreamingContext context)
         public static bool operator !=(BoundPeer left, BoundPeer right) =>
             Operator.Weave(left, right);
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BoundPeer"/> class from
+        /// comma-separated string.</summary>
+        /// <param name="peerInfo">A comma-separated string have format {pubkey},{host},{port}.
+        /// </param>
+        /// <returns>A <see cref="BoundPeer"/> from given data.</returns>
+        public static BoundPeer ParsePeer(string peerInfo)
+        {
+            if (peerInfo is null)
+            {
+                throw new ArgumentNullException(nameof(peerInfo));
+            }
+
+            string[] tokens = peerInfo.Split(',');
+            if (tokens.Length != 3)
+            {
+                throw new ArgumentException(
+                    $"'{peerInfo}', should have format <pubkey>,<host>,<port>");
```

Comment:
`ArgumentException` also has a constructor to receive `paramName`. How about passing it also?

```suggestion
                throw new ArgumentException(
                    $"'{peerInfo}', should have format <pubkey>,<host>,<port>", nameof(peerInfo));
```